README.md

```diff
@@ -85,7 +85,7 @@ $ ./unittest
 
 * centos 6/7
 * ubuntu
-* macosx
+* macos
```

Comment:
I don't think this is a typo. At least you can add reference about the usage clarification.

Reply:
we can see the history in [wikipedia](https://en.wikipedia.org/wiki/MacOS)